karma.conf.coverage.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env node */
```

Comment:
Why do we need to do the coverage separately? Can't we run test with coverage enabled and then collect coverage data?